rimage/image_processing_test.go

```diff
@@ -224,9 +227,7 @@ func TestCannyBlocks(t *testing.T) {
 		magDataInt[idx] = NewColor(uint8(math.Round(magData[idx])), uint8(math.Round(magData[idx])), uint8(math.Round(magData[idx])))
 	}
 	magOut := Image{
-		data:   magDataInt,
-		width:  gtGrad.Width(),
-		height: gtGrad.Height(),
```

Comment:
[q] curious what removing the height and width info from the Image did - did it speed things up somehow?

Reply:
the fields are unused. we only ever refer to magOut.data. not sure why the struct is used to wrap it and not just create only data 🤷